src/movies.js

```diff
@@ -1,15 +1,107 @@
 // Iteration 1: All directors? - Get the array of all directors.
+function getAllDirectors(array){
+    let directors = array.map((movies) => {
+    return movies.director
+    })
+ return directors
+}
+getAllDirectors(movies)
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
+
+
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
 
+function howManyMovies(array) {
+    let dramaMoviesSpielberg = array.filter((movies) =>{
+         return (movies.director === "Steven Spielberg") && (movies.genre.includes("Drama"))       
+    })
+    return dramaMoviesSpielberg.length
+}
```

Comment:
Good solution, very short , both conditions in the same line and using .includes, awesome!